packages/app-project/src/screens/ProjectAboutPage/ProjectAboutPage.js

```diff
@@ -1,12 +1,100 @@
-import React from 'react'
+import { Box, Grid, Heading } from 'grommet'
 import { arrayOf, bool, object, shape, string } from 'prop-types'
+import styled, { withTheme } from 'styled-components'
+import { withResponsiveContext } from '@zooniverse/react-components'
 
-function ProjectAboutPage ({ aboutPageData, inBeta, teamArray }) {
+/** Components */
+import StandardLayout from '@shared/components/StandardLayout'
+import { SpacedHeading, SpacedText } from '@zooniverse/react-components'
+import AboutSidebar from './components/AboutSidebar'
+import AboutDropdownNav from './components/AboutDropdownNav'
+import TeamMember from './components/TeamMember'
+import AboutMarkdownz from './components/AboutMarkdownz/AboutMarkdownz'
 
-  const { title = '' } = aboutPageData
+const PageHeading = styled(Heading)`
+  font-weight: normal;
+`
+
+const SidebarHeading = styled(SpacedHeading)`
+  padding: 5px 20px;
+`
+
+function ProjectAboutPage({
+  aboutPageData,
+  inBeta,
+  projectDisplayName,
+  teamArray,
+  screenSize
+}) {
+  const { content = '', title = '' } = aboutPageData
+
+  const isTeamPage = title.toLowerCase().includes('team')
 
   return (
-    <div>This is the {title} Page</div>
+    <StandardLayout inBeta={inBeta}>
+      <Box
+        background="neutral-6"
+        border={{
+          color: 'light-3',
+          size: '1px',
+          style: 'solid',
+          side: 'vertical'
+        }}
+        height={{ min: '98vh' }}
+        margin={{ left: 'large', right: 'large' }}
+        width={{ min: 'fill-available', max: 'xxlarge' }}
+        pad="large"
+        alignSelf="center"
+        flex
+      >
+        <Grid
+          columns={screenSize === 'small' ? ['auto'] : ['small', 'flex']}
+          gap={screenSize === 'small' ? '' : '8%'}
+        >
+          {screenSize !== 'small' ? (
+            <Box>
+              <SidebarHeading children="About" />
+              <AboutSidebar />
+            </Box>
+          ) : (
+            <AboutDropdownNav />
+          )}
+          <Box>
+            <PageHeading
+              children={isTeamPage ? 'The Team' : title}
+              weight="normal"
```

Comment:
```suggestion
              level='2'
              weight="normal"
```

I'd make this a H2 since the project name is H1 on the page and best practice is to have a single H1 per page.